bach/bach/dataframe.py

```diff
@@ -3329,7 +3329,7 @@ def unstack(
                 new_column_name = f'{column}__{curr_col}'
                 new_columns.append(new_column_name)
 
-                df[new_column_name] = None
+                df[new_column_name] = df[curr_col].from_value(base=df, value=None)
```

Comment:
I also considered to introduce some special object `UnassignedSeries` , that wouldn't require a `base_node` and that we would get from the DataFrame on assignment. Then we could write this perhaps as

```suggestion
                df[new_column_name] = df[curr_col].from_value(value=None)
```

But that change was a bit bigger and more impactful, and the only gain is that the user can drop the `base=df` part in function calls like this, so I threw that out.

Reply:
Can't we do the `from_value` in `DataFrame.__set_item__` when the value is None?

Reply:
For new columns we wouldn't know the dtype. We wouldn't know whether to call `SeriesString.from_value()` or `SeriesInt64.from_value()`